### PR TITLE
[11.0][ADD] purchase_request_usage_department

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
This module makes easier for users to create purchase requests for consumables. When creating purchase request, the product will be taken from the deafult usage of the department. That way users don't have to worry about finding the exact generic product shared by many departments (those products are generic and it is a pain to search among all the stockable ones)

Dependencies are now merged!!! :tada: :tada: :tada: :tada: :tada: :tada:

### CODE REVIEW

File: purchase_request_usage_department/models/purchase_request.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+from odoo import api, models
+
+
+class PurchaseRequestLine(models.Model):
+    _inherit = 'purchase.request.line'
+
+    @api.onchange('department_id', 'usage_id')
+    def onchange_department_id(self):
+        if self.department_id.usage_id:
+            self.usage_id = self.department_id.usage_id
```
Comment (sergio-teruel) [inline purchase_request_usage_department/models/purchase_request.py]: If you delete the department the usage would have the incorrrect value

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Review by HviorForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Comment by JordiBForgeFlow** [comment]
@aheficent can you resolve the conflicts?

**Comment by LoisRForgeFlow** [comment]
@aheficent and have a look at travis? :smile:

**Comment by AaronHForgeFlow** [comment]
Ready to merge IMHO

**Pull request merged**